.github/workflows/ci-cleanup.yml

```diff
@@ -0,0 +1,55 @@
+run-name: Cleanup ${{ github.head_ref }}
+on:
+  pull_request_target:
+    types: labeled
+    paths:
+      - 'images/**'
+
+jobs:
+  clean_ci:
+    name: Clean CI runs
+    runs-on: ubuntu-latest
+    permissions:
+      actions: write
+    steps:
+      - env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Should we change the name "GITHUB_TOKEN" for the secret?  According to documentation "Secret names must not start with GITHUB_".

Reply:
@Alexey-Ayupov it is the automated token. I.e. not manual I believe  https://docs.github.com/en/actions/security-guides/automatic-token-authentication#about-the-github_token-secret